data/org_cluster_map_handler.py

```diff
@@ -0,0 +1,132 @@
+    locations_source_path = f'{os.getcwd()}/{output_dir}/org-locations.js'
+    locations_destination_path = f'{os.getcwd()}/static/org-locations.js'
+    os.rename(locations_source_path, locations_destination_path)
+
+    # Make changes in map.html to support django syntax with needed CSS
+    with open('{}/map.html'.format(output_dir)) as f:
+        django_supported_htmls = []
+        for html_line in f:
+            line = html_line.strip()
+            if line.__contains__('<html>'):
+                django_supported_htmls.append('{% load staticfiles %}\n')
+                django_supported_htmls.append(line + '\n')
+            elif line.__contains__('</head>'):
+                adjust_prop = '''
+                <style>
+                    #map {
+                    width: 60%;
+                    height: 300px;
+                    margin: auto;
+                    box-shadow: 0px 0px 25px 2px;
+                    }
+                    @media only screen and (max-width:750px){
+                        #map {
+                            width: 90%
+                        }
+                    }
+                </style>\n
+                '''
+                meta_charset = '<meta charset="utf-8">'
+                adjust_prop = adjust_prop.strip().replace('    ', '')
+                django_supported_htmls.insert(6, meta_charset + '\n')
+                django_supported_htmls.append(adjust_prop + '\n')
+                django_supported_htmls.append(line + '\n')
+            elif line.__contains__('https://'):
+                line = line.replace('https:', '').replace(' />', '>')
+                django_supported_htmls.append(line.strip() + '\n')
+            elif line.__contains__('<link '):
+                line = line.replace('href="', 'href="{% static \'')
+                line = line.replace('.css', '.css\' %}').replace(' />', '>')
+                django_supported_htmls.append(line + '\n')
+            elif line.__contains__('<script '):
+                line = line.replace('src="', 'src="{% static \'')
+                line = line.replace('.js', '.js\' %}')
+                if line.__contains__(' type="text/javascript"'):
+                    line = line.replace(' type="text/javascript"', '')
+                django_supported_htmls.append(line + '\n')
+            elif line.__contains__('Mouse over') or len(line) == 0:
+                pass
+            else:
+                django_supported_htmls.append(line + '\n')
+
+    # Remove and Add 'map.html' to django templates directory
+    html_map_path = '{}/{}/map.html'
+    html_map_source_path = html_map_path.format(os.getcwd(),
+                                                output_dir)
+    html_map_destination_path = html_map_path.format(os.getcwd(),
+                                                     'templates')
+    os.remove(html_map_source_path)
+    with open(html_map_destination_path, 'w+') as f:
+        f.writelines(django_supported_htmls)
```

Comment:
can we use [this solution](https://stackoverflow.com/questions/39086/search-and-replace-a-line-in-a-file-in-python) for this whole process? If yes, can you figure out which one is better and why, and go with it.

Reply:
Sure! I will check the solution 🙂

Reply:
Going with the suggested solution :+1: